code/modules/mob/living/carbon/carbon.dm

```diff
@@ -543,6 +547,10 @@
 			see_invisible = G.invis_override
 		else
 			see_invisible = min(G.invis_view, see_invisible)
+		if(!isnull(G.lighting_alpha) && hud_used)
+			var/obj/screen/plane_master/lighting/L = hud_used.plane_masters["[LIGHTING_PLANE]"]
+			if (L)
+				L.alpha = G.lighting_alpha

```

Comment:
This overrides eye lighting vision? Seems kind of odd, but okay.
Also, does this get reset anywhere? Like when you take glasses off, it doesn't seem to actually reset the lighting plane's alpha.